subprojects/ide-native/src/main/java/org/gradle/ide/xcode/plugins/XcodePlugin.java

```diff
@@ -141,7 +141,7 @@ private GenerateXcodeWorkspaceFileTask getWorkspaceTask() {
 
     private void includeBuildFileInProject(Project project) {
         if (project.getBuildFile().exists()) {
-            xcode.getProject().addSourceFile(project.getBuildFile());
+            xcode.getProject().getGroups().getRoot().from(project.getBuildFile());
```

Comment:
Will anything not explicitly included just get ignored?  If project.getBuildFile() doesn't exist, does xcode complain?

What about other build-like files?  build.gradle.kts, settings.gradle(.kts), gradle.properties and any other applied .gradle files?

I'm not saying we should do anything about those, but maybe a TODO here explaining that this isn't complete.

Reply:
Sounds like a good idea. I will add a TODO.